hack/packages/package-tools/cmd/package-bundle-push.go

```diff
@@ -27,7 +27,6 @@ var packageBundlePushCmd = &cobra.Command{
 
 func init() {
 	packageBundleCmd.AddCommand(packageBundlePushCmd)
-	packageBundlePushCmd.Flags().StringVar(&packageRepository, "repository", "", "Package repository of the package bundle being pushed")
```

Comment:
same as above, it would keep it optional

Reply:
I agree. It's an option now.